com.unity.ml-agents/Runtime/MLAgentsSettings.cs

```diff
@@ -0,0 +1,38 @@
+using UnityEngine;
+
+namespace Unity.MLAgents
+{
+    class MLAgentsSettings : ScriptableObject
+    {
+        [SerializeField]
+        private bool m_ConnectTrainer = true;
```

Comment:
There were other settings that we talked about adding later - should we group/nest these now so that we don't have to go through hoops to move them around in the future?

Reply:
I can group them into **Training Port**.

Reply:
@chriselion by group/nest these do you mean nesting them in the code (like a subclass in MLAgentsSettings) or just grouping in editor?

Reply:
Either way - I think using a subclass in MLAgentsSettings makes the grouping in the editor more automatic.

Reply:
I guess if we always need to manually layout the fields in `MLAgentsSettingsProvider.OnGUI`, it's less important. But it might let us just add the "top level" settings (like TrainerSettings) and not have to worry about new fields when we add them.